specification/sphere/Sphere.Management/examples/2024-04-01/Catalogs_CountDevices.json

```diff
@@ -1,12 +1,12 @@
 {
   "operationId": "Catalogs_CountDevices",
-  "title": "Catalogs_CountDevices",
   "parameters": {
```

Comment:
Can we avoid the changes in the examples which is most ordering change. This would reduce the changeset.